rest-api-spec/src/main/resources/rest-api-spec/test/nodes.reload_secure_settings/10_basic.yml

```diff
@@ -1,5 +1,8 @@
 ---
 "node_reload_secure_settings test wrong password":
+  - skip:
+      version: " - 7.9.99"
```

Comment:
Will adjust this to 7.6.99 once https://github.com/elastic/elasticsearch/pull/55060 and https://github.com/elastic/elasticsearch/pull/55059 are merged

Reply:
I think you'll want to make the version here `7.99.99` otherwise this will start failing when we branch for `7.10`.

Reply:
well @mark-vieira , 7.10 will have the changes so it would be safe to run this against it  ( as said I will adjust this to 7.6.99 within the next couple of hours )

Reply:
Ah, if we plan to backport soon then disregard.